cmd/tesseract/README.md

```diff
@@ -6,4 +6,190 @@ This directory contains TesseraCT binaries for [AWS or Vanilla S3+MySQL](./aws/)
 It also contains Dockerfiles to build these binaries, and to
 bundle them with the CA roots they accept.
 
-TODO: move /docs/configuration.md into here
+## TesseraCT configuration
+
+This file explains how to configure TesseraCT binaries.
+
+It contains two main parts:
+
+ 1. [Chain lifecyle](#chain-lifecycle): with settings impacting how submission
+ are processed
+ 2. [Setup](#setup): with instructions on how to setup TesseraCT resources
+
+You can find information about implementation-specific settings in each
+corresponding subdirectory.
+
+### Chain lifecycle
+
+When TesseraCT receives a new chain submssion, it is first
+[filtered](#chain-filtering), and is then sent to the Tessera library where it goes
+through the [Append Lifecycle](#append-lifecycle).
+The chain may be [deduplicated using TesseraCT's and Tessera's antispam features](#antispam).
+
+#### Chain filtering
+
+Chains accepted by TesseraCT can be filtered by setting the following flags:
+
+- `root_pem_file`: Path to a file containing root certificates that are
+acceptable to the log. Chains MUST be signed by one of the roots included in
```

Comment:
Might be worth flipping the sense around on all these bullets so they all primarily talk in terms of rejecting (or accepting if you prefer)?

```suggestion
acceptable to the log. Chains not originating from one of the roots in this file will be rejected.
```